web/pages/fast.svelte

```diff
@@ -14,9 +14,17 @@
   import Loader from '../ui/loader.svelte'
   import PostCard from '../ui/post-card.svelte'
   import TwoStepsPage from '../ui/two-steps-page.svelte'
+
+  let twoStepsPageInstance: TwoStepsPage
+
+  fastPosts.subscribe(() => {
```

Comment:
Every time you subscribe to some store think when you will unsubscribe from it.

Right now every time user visited `fast` page they will subscribe again and old subscription will be still active.